kuma/banners/jinja2/banners/banners.html

```diff
@@ -0,0 +1,52 @@
+{% set banners = get_banners() %}
+<div
+    id="developer_needs"
```

Comment:
I'm working on the next iteration of this which will support multiple banners. This `id` will be removed as part of that.

Reply:
I'm guessing this will be a database field or use a slugified version of the name. If you intend to add a new database field, I think we should do it with this PR, which already has a migration.

Reply:
I don't like that this div tag gets injected all and every time even though there are potentially, no banners. 
A big step forward would be to move the `{% if banners %}` statement on line 1.

Reply:
Same here, why does it need an ID?